src/TodoListItem.module.css

```diff
@@ -0,0 +1,109 @@
+@import url('https://fonts.googleapis.com/css?family=Sofia');
+*,
+*::before,
+*::after {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+}
+
+body {
+    background-color: #efefef;
+}
+
+h1 {
+    font-family: 'sofia', sans-serif;
+    color: #09a8e2; 
+}
+
+ul {
+    list-style: none;
+}
```

Comment:
I think these general styles would fit better in your `index.css`.

You should probably think and use `.module.css` files as if they can only contain styles that only affect the component, or "module", that they're specifically for.